compiler/noirc_evaluator/src/ssa/parser/mod.rs

```diff
@@ -796,6 +826,18 @@ impl<'a> Parser<'a> {
         }
     }
 
+    fn eat_byte_str(&mut self) -> ParseResult<Option<String>> {
+        if matches!(self.token.token(), Token::ByteStr(..)) {
+            let token = self.bump()?;
```

Comment:
```suggestion
            // skip the leading 'b'
            let token = self.bump()?;
```

Reply:
Oh, this is actually consuming the current token which is the byte string. The other methods (for example `eat_str`) are the same.